### PR TITLE
Added files with a popup for review

### PR DESCRIPTION
**Pull request by Anatolii-Petrenko**
### HTML-CSS-Popup

[Demo](https://anatolii-petrenko.github.io/popup-task-html-css/#)
[Code base](https://github.com/Anatolii-Petrenko/popup-task-html-css)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Anatolii-Petrenko** [comment]
> 

Сhecked that haven't followed [common mistakes](https://kottans.org/documentation/docs/doc/code-review/#popup-task).

Please, review :)

File: submissions/Anatolii-Petrenko/html-css-popup/css/reset.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/* Reset Styles */
```
Comment (A-Ostrovnyy) [inline submissions/Anatolii-Petrenko/html-css-popup/css/reset.css:1]: Your PR should contain files for review, the file `reset.css` is not for review. remove it.

File: submissions/Anatolii-Petrenko/html-css-popup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+::-webkit-scrollbar-thumb:hover {
+	background: rgb(150, 150, 150);
+}
+
+@media (max-width: 540px) {
+	.popup__body{
+		right: calc(50% - calc(320px / 2));
+	}
+}
```
Comment (A-Ostrovnyy) [inline submissions/Anatolii-Petrenko/html-css-popup/css/style.css:194]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/Anatolii-Petrenko/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+</head>
+
+<body>
+	<div class="wrapper">
+		<header class="header">
+			<nav class="menu">
+				<ul class="menu__list">
+					<li>
+						<a class="menu__item" href="#">
+							<img class="icon" src="img/header/mail.png" alt="Mail">
```
Comment (A-Ostrovnyy) [inline submissions/Anatolii-Petrenko/html-css-popup/index.html:20]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome design!

File: submissions/Anatolii-Petrenko/html-css-popup/css/reset.css (hunk 1 above)
Comment (Anatolii-Petrenko) [reply submissions/Anatolii-Petrenko/html-css-popup/css/reset.css]: Done.

File: submissions/Anatolii-Petrenko/html-css-popup/css/style.css (hunk 2 above)
Comment (Anatolii-Petrenko) [reply submissions/Anatolii-Petrenko/html-css-popup/css/style.css]: Done in commit https://github.com/kottans/frontend-2022-homeworks/commit/05b85e7406a9d6b85dba52496bc7d0cbb506eadf.

File: submissions/Anatolii-Petrenko/html-css-popup/index.html (hunk 3 above)
Comment (Anatolii-Petrenko) [reply submissions/Anatolii-Petrenko/html-css-popup/index.html]: Alt attribute values for all images in html file were replaced with empty values in commit https://github.com/kottans/frontend-2022-homeworks/commit/05b85e7406a9d6b85dba52496bc7d0cbb506eadf.

**Comment by Anatolii-Petrenko** [comment]
Please, re-review :)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**